services/settings/pkg/store/defaults/permissions.go

```diff
@@ -356,15 +413,15 @@ func SpaceAbilityPermission(c settingsmsg.Permission_Constraint) *settingsmsg.Se
 	}
 }
 
-// WritePublicLinkPermission is the permission to write public links
-func WritePublicLinkPermission(c settingsmsg.Permission_Constraint) *settingsmsg.Setting {
+// WriteFavoritesPermission is the permission to mark/unmark files as favorites
+func WriteFavoritesPermission(c settingsmsg.Permission_Constraint) *settingsmsg.Setting {
 	return &settingsmsg.Setting{
-		Id:          "11516bbd-7157-49e1-b6ac-d00c820f980b",
-		Name:        "PublicLink.Write",
```

Comment:
This removes the `PublicLink.Write` permission. Do we really want that?

Reply:
No we don't. But isn't it still here: https://github.com/owncloud/ocis/pull/7700/files#diff-6dad214221d8e9023e7d9a8a99a9d9446553d5c8c97be4d877274c49777e9c0aR70-R87 ?

Reply:
Ah, sorry. Overlooked that.